Tests/kaas/kaas-sonobuoy-tests/README.md

```diff
@@ -61,26 +60,30 @@ For test development with Sonobuoy, [KinD](https://kind.sigs.k8s.io/) is used as
     sonobuoy status
```

Comment:
I don't think we need to use the `sonobuoy status` command, as we are now using the `--wait` flag in the `make dev-run` command. I would say `sonobuoy status` and the related text above apply when the user/developer is using the `make dev-run-background` command now.

Reply:
That's right. I removed the `sonobuoy status` commands.

Reply:
This command is also mentioned in the README of this directory. Will the README be updated in this PR, or will it be part of https://github.com/SovereignCloudStack/docs/pull/221?